ios/sistema/Info.plist

```diff
@@ -85,8 +85,22 @@
 		<string>Mulish-Medium.ttf</string>
 		<string>Mulish-MediumItalic.ttf</string>
 		<string>Mulish-Regular.ttf</string>
-		<string>Mulish-SemiBold.ttf</string>
-		<string>Mulish-SemiBoldItalic.ttf</string>
+		<string>Roboto-Regular.ttf</string>
+		<string>AntDesign.ttf</string>
+	  <string>Entypo.ttf</string>
+	  <string>EvilIcons.ttf</string>
+	  <string>Feather.ttf</string>
+	  <string>FontAwesome.ttf</string>
+	  <string>FontAwesome5_Brands.ttf</string>
+	  <string>FontAwesome5_Regular.ttf</string>
+	  <string>FontAwesome5_Solid.ttf</string>
+	  <string>Foundation.ttf</string>
+	  <string>Ionicons.ttf</string>
+	  <string>MaterialIcons.ttf</string>
+	  <string>MaterialCommunityIcons.ttf</string>
+	  <string>SimpleLineIcons.ttf</string>
+	  <string>Octicons.ttf</string>
+	  <string>Zocial.ttf</string>
```

Comment:
I think they came in from merge conflicts, I accepted incoming because I didn't know where they came from but just in case other people were using it.

Reply:
Hm I've looked over most people's PRs and I don't think anyone's using this stuff. Well if you can delete them and it doesn't break anything I'd get rid of them haha

Reply:
oh it was bc it was breaking the mui dependency and the build if these weren't in the info.plist so i added them. if no one's using the mui dependency then for sure let's remove them. mb!